lib/api/src/grpc/proto/points_internal_service.proto

```diff
@@ -22,6 +23,20 @@ service PointsInternal {
   rpc Get (GetPointsInternal) returns (GetResponse) {}
 }
 
+
+message SyncPoints {
+  string collection_name = 1; // name of the collection
+  optional bool wait = 2; // Wait until the changes have been applied?
+  repeated PointStruct points = 3;
+  optional PointId from_id = 4; // Start of the sync range
+  optional PointId to_id = 5; // End of the sync range
+}
+
+message SyncPointsInternal {
```

Comment:
Will there be some kind of public `SyncPoints` API?

Reply:
no

Reply:
Do we need 2 structures here then?

Reply:
it is consistent with other API in this way